donate/settings/base.py

```diff
@@ -0,0 +1,386 @@
+import django
+import logging.config
+from configurations import Configuration
+
+from .environment import (
+    env,
+    app,
+    root
+)
+
+
+class Base(Configuration):
+    WAGTAIL_SITE_NAME = "donate"
+    RECAPTCHA_ENABLED = False
+
+    ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+
+    @property
```

Comment:
these `@property` decorated methods act as computed properties for django settings that need to be evaluated when accessed, or after class declaration.